packages/roosterjs-react-emoji/lib/components/EmojiPane.tsx

```diff
@@ -31,13 +30,17 @@ export interface EmojiPaneProps {
     partialListClassName?: string;
 }
 
-export default class EmojiPane extends React.Component<EmojiPaneProps, EmojiPaneState> {
+export interface InternalEmojiPaneProps extends EmojiPaneProps {
```

Comment:
Does this need to be exported? Or you are making this Internal by not including it in the bundle?

Reply:
It needs to be exported here so the Plugin can use, but it's not exported in the index file.

The design is to allow you to add custom class names, onSelect depends on Emoji and that depends on a bunch of other interfaces. Right now, we don't need to expose onSelect as it's currently only being used by the emoji plugin.